documentation/modules/configuring/con-config-kafka.adoc

```diff
@@ -21,18 +21,14 @@ Configuration options that are particularly important include the following:
 * Rack awareness
 * Metrics
 * Cruise Control for cluster rebalancing
+* Metadata version for KRaft-based Kafka clusters
+* Inter-broker protocol version for ZooKeeper-based Kafka clusters
 
-For a deeper understanding of the Kafka cluster configuration options, refer to the link:{BookURLConfiguring}[Strimzi Custom Resource API Reference^].
-
-.Kafka versions
-
-The `inter.broker.protocol.version` property for the Kafka `config` must be the version supported by the specified Kafka version (`spec.kafka.version`).
-The property represents the version of Kafka protocol used in a Kafka cluster.
+The `metadataVersion` or `inter.broker.protocol.version` property for the Kafka `config` must be a version supported by the specified Kafka version (`spec.kafka.version`).
+The property represents the Kafka metadata or inter-broker protocol version used in a Kafka cluster.
+If either of these properties is not set in the configuration, the Cluster Operator updates the version to the default for the Kafka version used.  
```

Comment:
Should we maybe add here that:
* The oldest supported metadata version is `3.3`
* Using an older metadata version than the Kafka version might cause some KAfka features to be disabled?